.gitignore

```diff
@@ -70,6 +70,7 @@ typings/
 
 # dotenv environment variables file
 .env
+.envrc
```

Comment:
Good catch! I committed an `.env.development` file in a personal project recently; luckily it was empty.